### ISSUE TITLE
File names containing spaces cause template_categorical_variables() crash

### ISSUE THREAD

**Issue by earnaud** [EDIorg/EMLassemblyline#25]
I am just trying out the EML Assembly Line on an external use case, and it appears that the file I am trying to use is not recognized. 

In fact, at line 154 of `template_categorical_variables.R`, a regular expression uses the `[:graph:]` label which do not recognize the " " present in the filenames. Thus, EML-AL is lost and returns me the following error:
```error
No categorical variables found.
Error in x$template[[attribute_files[i]]] : 
  attempt to select less than one element in get1index
```

**Comment by earnaud** [comment]
A solution would be, before the l.154 chunk, to use `gsub()` to replace every space by underscore. E.g. :
```r
names(x$template) <- gsub(" ", "_", names(x$template))
```

**Comment by clnsmth** [comment]
This has been fixed and merged into the development branch (3e34bba86e29ead878da2218338355e4c0dfd936).
